_includes/selector.html

```diff
@@ -728,6 +654,11 @@
                 if (this.active_release === "Nightly" && method === "pip") isDisabled = true;
                 return isDisabled;
             },
+            disableUnsupportedPackage(package) {
+                var isDisabled = false;
+                if (package === "cuSignal" && ((this.active_cuda_ver.startsWith("12") && this.active_method === "Conda") || this.active_release === "Nightly")) isDisabled = true;
+                return isDisabled;
+            },
```

Comment:
Is this logic working as expected?

I'm still able to select `cusignal` with conda, CUDA 12, and nightly selected.

![image](https://github.com/rapidsai/docs/assets/7400326/337362df-2406-45c4-bb73-0a2143a7e968)

Reply:
oooo no, will debug - nice catch.

Reply:
Found the issue - when I added in the second line for specific package selection, I failed to add the disabled info to the options-container class.

Resolved in https://github.com/rapidsai/docs/pull/414/commits/a854dbe4cd82a8a7efd2e2c293a08082f89f84ac